runtime/src/accounts_db.rs

```diff
@@ -1609,20 +1612,44 @@ impl AccountsDb {
 
     // Construct a vec of pubkeys for cleaning from:
     //   uncleaned_pubkeys - the delta set of updated pubkeys in rooted slots from the last clean
-    //   zero_lamport_pubkeys - set of all alive pubkeys containing 0-lamport updates
+    //   dirty_stores - set of stores which had accounts removed
```

Comment:
Nit: maybe this comment would also be helpful here: https://github.com/solana-labs/solana/pull/17601/files#diff-1090394420d51617f3233275c2b65ed706b35b53b115fe65f82c682af8134a6fR851 when the `dirty_stores` field is declared in the struct.

Also, `dirty_stores` is added to in `add_root`, when accounts haven't technically been "removed" yet: https://github.com/solana-labs/solana/pull/17601/files#diff-1090394420d51617f3233275c2b65ed706b35b53b115fe65f82c682af8134a6fR5340. Maybe a more apt description is something along the lines of

 "set of stores which contain updates to accounts that:

1) Outdate earlier updates to earlier stores
2) Were set to zero-lamports in later stores, and are now potentially free for cleaning"